gatsby-config.js

```diff
@@ -53,7 +53,69 @@ module.exports = {
         trackingId: `UA-130227707-1`,
       },
     },
-    `gatsby-plugin-feed`,
+    {
+      resolve: `gatsby-plugin-feed`,
+      options: {
+        query: `
+          {
+            site {
+              siteMetadata {
+                title
+                description
+                siteUrl
+                site_url: siteUrl
+              }
+            }
+          }
+        `,
+        feeds: [
+          {
+            serialize: ({ query: { site, allMarkdownRemark } }) => {
```

Comment:
There are quite a few options that could be added to this, but for now, this is just the basic/default with the addition of a postscript that includes a clickable link to the article on the site